general/contributing/development.md

```diff
@@ -135,3 +135,43 @@ When submitting a new PR, please ensure you do the following things. If you have
 * Expect review and discussion. If you can't back up your changes with a good description and through review, please reconsider whether it should be done at all. All PRs to `dev` require at least one approving review from an administrative team member, however we welcome and encourage reviews from any contributor, especially if it's in an area you are knowledgeable about. More eyes are always better.
 
 * All PRs require review by at least two team members before being merged into `master`, though reviews from any contributor are welcome! After the second team member review the PR may be merged immediately, or more review or feedback requested explicitly from other contributors if required.
+
+## Building and Testing Inside a Docker Container
+
+We need to install all development dependencies and pull down the code inside the container before we can compile and run.
+
+> [!NOTE]
+> Run each commmand on a separate line.
+> The container we'll test in is named `jftest`
+> Within Docker, anytime the entrypoint executable is terminated, the session restarts, so just exec into it again to continue.  This is also why we explicitly kill it to reload the new version.
```

Comment:
I'd say these should all be on one line. Also, only use one space before a new sentence.